test/elements/test_folder.py

```diff
@@ -108,8 +108,11 @@ def test_update_folder(self, m):
                 "version": 1,
             },
         )
-        folder = self.grafana.folder.update_folder(title="Departmenet DEF", uid="nErXDvCkzz", version=1, overwrite=True)
+        folder = self.grafana.folder.update_folder(
+            title="Departmenet DEF", uid="nErXDvCkzz", new_uid="nErXDvCkzz", version=1, overwrite=True
```

Comment:
Maybe we can use a different value for the `new_uid` argument here in order to better demonstrate its purpose?
```suggestion
            title="Departmenet DEF", uid="nErXDvCkzz", new_uid="oFsYEwDlaa", version=1, overwrite=True
```